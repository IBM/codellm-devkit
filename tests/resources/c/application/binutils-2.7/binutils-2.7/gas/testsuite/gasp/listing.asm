

	.HEADING	" ""QUOTE"" "
	.PAGE
	.PRINT LIST
	foo		
	.PRINT NOLIST
	foo


	.FORM	LIN=12
	.FORM	COL=90
	.FORM	LIN=123 COL=23
	
	.END
