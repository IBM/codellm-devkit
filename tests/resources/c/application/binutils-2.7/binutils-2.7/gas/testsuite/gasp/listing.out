!

!

!	.HEADING	" ""QUOTE"" "
	.title	" "QUOTE" "
!	.PAGE
	.eject
!	.PRINT LIST
	.list
!	foo		
	foo		
!	.PRINT NOLIST
	.nolist
!	foo
	foo
!

!

!	.FORM	LIN=12
	.psize 12,132
!	.FORM	COL=90
	.psize 60,90
!	.FORM	LIN=123 COL=23
	.psize 123,23
!	
!	.END
